scripts/ec2/tags.coffee

```diff
@@ -1,26 +1,57 @@
-
-cson = require 'cson'
 ec2 = require('../../ec2.coffee')
-util = require 'util'
+moment = require('moment')
+validator = require('validator')
+
 
-# Expose to other scripts
 tags =
 
-  addSchedule: (msg, instances, schedule="8:18") ->
-    ec2.createTags {Resources: instances, Tags: [Key: "Schedule", Value: schedule]}, (err, res) ->
-      if err
-        console.log "Error creating tags: #{err}"
-        if msg
-          return msg.send "Error creating tags: #{err}" if err
+  DEFAULT_SCHEDULE: "8:18"
+
+  SCHEDULE_TAG: "RunSchedule"
+
+  # convert our concise schedule strings ("8:18") to the new UTC-based format
+  formatSchedule: (hours) ->
+    [_, start, stop] = hours.match /^(\d{1,2}):(\d{1,2})$/
+
+    invalidHours = [start, stop]
+      .map((e) -> validator.isInt(e, {min: 0, max: 24}))
```

Comment:
technically should probably be `0` to `23`, but other parts of this module allow `24`, so we will stick with it; `moment` appears to treat `24` as `0`